### DISCUSSION TITLE
Trivy misconfiguration support for Azure

### DISCUSSION THREAD

**Discussion by pritam-yadu** [aquasecurity/trivy#4018] [Q&A] [answered]
Hi Team, 

I see Trivy currently does not support scanning misconfigurations in Azure. Are there any plans to support that? If yes, how long will it take(ballpark figure)? Just to clarify, I am interested in misconfigurations in Azure services and not container image scans.

Thanks,
Pritam Yaduvanshi

**Comment by itaysk** [comment, marked as answer]
trivy supports scanning ARM IaC templates but not connecting to Azure API. this is in scope for Trivy but we don't have a date to share. i don't expect it to happen soon

**Reply by pritam-yadu** [reply]
Thanks @itaysk .
